code/game/turfs/flooring/flooring_premade.dm

```diff
@@ -59,7 +59,9 @@
 	icon_state = "wood"
 	initial_flooring = /decl/flooring/wood
 
-
+/turf/simulated/floor/wood/Initialize()
+	icon_state = pick("wood", "wood1", "wood2")
+	return ..()
```

Comment:
This randomizes how the wood looks on every tile? Does it fit together nicely?

Reply:
I've downloaded and checked few times(restarted the round) branch to see the tiles and they were fine to me each time.